modules/hal_nxp/mcux/mcux-sdk-ng/drivers/drivers.cmake

```diff
@@ -202,6 +201,14 @@ if(CONFIG_SOC_FAMILY_MCXW OR CONFIG_SOC_MCXN947 OR CONFIG_SOC_MCXN547)
   set_variable_ifdef(CONFIG_SOC_FLASH_MCUX CONFIG_MCUX_COMPONENT_driver.flash_k4)
 endif()
 
+if(CONFIG_SOC_FAMILY_MCXW OR CONFIG_SOC_MCXN547)
```

Comment:
why or mcxn547?

Reply:
Typo, meant MCXN947, will be fixed in the next push.